params/network_config.go

```diff
@@ -70,6 +70,15 @@ type Network struct {
 	RelatedChainID         uint64          `json:"relatedChainId" validate:"omitempty"`
```

Comment:
there seems to be a bunch of stuff to support adding user networks, we should probably add a field indicating if a network is embedded/user-added (for example, to see if we enable deleting it)